fire/cli/mtl.py

```diff
@@ -330,8 +299,20 @@ def find_observationer(importér):
 
 
 # ------------------------------------------------------------------------------
+def find_observationer():
+    """Opbyg dataframe med alle observationer"""
```

Comment:
Den her doc-string må skulle afspejle denne kommentar "Hvis importér ikke er sat: Læs observationer direkte i projektregnearket" fra forrige commit, ikke?

Reply:
Ja - den beskriver det der faktisk sker, set fra overfladen af funktionskaldet, men det gør forskellen på de to funktioner uklar - finder lige på noget bedre, efter at have rettet et par absurde bommerter andetsteds :-)